File: lua/refactoring/tests/refactor/123/js/example/inline_var.start.js
// This the straight from the book (slight modifications)
function orderCalculation(order) {
    const basePrice = order.quantity * order.itemPrice;
    basePrice;
    return (
        basePrice -
        Math.max(0, order.quantity - 500) * order.itemPrice * 0.05 +
        Math.min(basePrice * 0.1, 100)
    );
}

File: lua/refactoring/tests/refactor/123/go/example/inline_var.start.go
package main

import "math"

type Order struct {
    quantity  float64
    itemPrice float64
}

func orderCalculation(order Order) float64 {
    basePrice := order.quantity * order.itemPrice
    return basePrice - math.Max(0, order.quantity-500)*order.itemPrice*0.05 + math.Min(basePrice*0.1, 100.0)
}
